lib/exec.js

```diff
@@ -21,6 +21,7 @@ module.exports = function({
 
     const p = new Promise((resolve, reject) => {
         const child = spawn(cmd, args, {
+            shell: true,
             ...opts,
             env: childEnv,
```

Comment:
Sort the props so we spread `..opts` first or last depending on the overrides?

Reply:
Yeah, `childEnv` is a combination of the `env` passed as a param and `process.env`, so it's fine to come after `...opts` (it won't override `opts.env` because we already deconstructed that out of the object).  It would also be safe for `...opts` to be last for the same reason, though.

If `opts.shell===false`, though, I would expect it to override the `true` default set here, so that's why I put it before `...opts`.

Reply:
So moving `...opts` to the last position should be fine, I'll do that